common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java

```diff
@@ -116,7 +118,19 @@ void processChanges(ScoreboardSegment segment) {
             }
         }
 
+        List<TerritoryAttackTimer> oldTimers = attackTimers;
         attackTimers = newList;
+
+        // Signal events for the new vars
+        for (TerritoryAttackTimer attackTimer : attackTimers) {
+            if (attackTimers.stream()
+                    .anyMatch(territoryAttackTimer ->
+                            territoryAttackTimer.territory().equals(attackTimer.territory()))) {
+                continue;
+            }
+
+            WynntilsMod.postEvent(new GuildWarQueuedEvent(attackTimer));
```

Comment:
Shouldn't this be comparing against oldTimers?

Reply:
Yeah, I couldn't test it when I wrote it..